tools/ci_build/build.py

```diff
@@ -1741,8 +1741,9 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                 executables.append("onnxruntime_global_thread_pools_test")
                 executables.append("onnxruntime_api_tests_without_env")
                 executables.append("onnxruntime_customopregistration_test")
-                for exe in executables:
```

Comment:
This prevents tests from running on non-Windows machines when `--ctest_path` is empty. Ex: Linux QNN CI Pipeline